examples/advection_1d/test_advection.py

```diff
@@ -4,7 +4,7 @@ def test_1d_advection():
 
     tests against expected classic, sharpclaw, and high-order weno results """
 
-    from . import advection_1d
+    import advection_1d
```

Comment:
Does this work in Python 2.x?

Reply:
so on my computer the test wouldn't run if i had the "from ." and i think that is the problem, on my computer, the tests run fine, but on Travis, I don't know what the outcome will be. Also, on Travis, the error norm produced for regular advection 1D is being off, when on my computer it is fine